packages/hierarchy-browser/src/utils/utils.ts

```diff
@@ -141,6 +145,35 @@ export function getStaticNeighborEntities(
 	}, [] as IEntityDetail[])
 }
 
+export function createEntityMap(
+	entities?: IEntityDetail[],
+): IEntityMap | undefined {
+	if (entities) {
+		return entities.reduce((acc, entity) => {
+			const id = entity.id
+			acc[id] = entity
+			return acc
+		}, {} as IEntityMap)
+	}
+}
+
+export function isEntitiesAsync(
```

Comment:
The name of this feels awkward. What does this function do? Maybe a jsdoc comment would help